puma/var_vs_var.py

```diff
@@ -157,6 +157,9 @@ def __init__(self, grid: bool = False, **kwargs) -> None:
             raise ValueError("Not more than one ratio panel supported.")
         self.initialise_figure()
 
+    def __nonzero__(self):
+        return bool(self.plot_objects)
```

Comment:
out of interest what s this line doing? wondering if its necessary given it's not hit by any of the tests